api_tests/dry/rfc003/test.js

```diff
@@ -79,6 +84,7 @@ describe("RFC003 HTTP API", () => {
                 alpha_ledger_refund_identity: "",
                 beta_ledger_redeem_identity: "",
                 alpha_ledger_lock_duration: 0,
+                peer_address: "0.0.0.0",
```

Comment:
Would prefer this be called just `peer` because I think that's what it will be called in the future once we have multiple ways of describing a peer.

Reply:
Yes agreed, I already see this field accepting formats like `"pubkey@address"` or even just `"pubkey"`